SE-13759: Calling Oracle Stored Procedure with TO_DATE() by DB Connec…

emariotti3 opened the pull request:
…tor 1.5.5 aborts 'Timestamp format must be yyyy-mm-dd hh:mm:ss[.fffffffff]'

Commit:
SE-13759: Calling Oracle Stored Procedure with TO_DATE() by DB Connector 1.5.5 aborts 'Timestamp format must be yyyy-mm-dd hh:mm:ss[.fffffffff]'

Bardammu reviewed (commented):
Please ask the PM to review this PR

estebanwasinger commented on modules/ROOT/pages/db/database-connector-examples.adoc:
```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
+Database connector supports the use of callable statements with the format { call procedureName(:param1, :param2, ..., :paramN) } where parameters will be matched positionally, that is to say, :paramN will match the Nth parameter in the stored procedure declaration. Suppose you have declared a stored procedure which receives two parameters: myFirstParameter and mySecondParameter, in that order. T ...
+
+It also does not support the use of engine specific embedded functions, for example, you might want to change a date to a specific format before calling a procedure in an Oracle database, however, the following approach will not work:  { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }. All data transformations should be done using dataweave, before the invocation to ...
+
```
This new text should have a header saying something like "Using dates on Stored Procedures".

kevintroller replied:
Updated.

estebanwasinger commented on modules/ROOT/pages/db/database-connector-examples.adoc:
```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
+		    <db:output-parameter key="result" type="VARCHAR"></db:output-parameter>
+		</db:output-parameters>
+	</db:stored-procedure>
+----
```
How a customer can test this example?
Is there a way to provide the store procedure code?

kevintroller replied:
Resolved in commit [59a4c4c](https://github.com/mulesoft/docs-connectors/pull/508/commits/59a4c4c4a212175fa1612625d933c9901ebd1c9a)

estebanwasinger commented on modules/ROOT/pages/db/database-connector-examples.adoc:
```diff
@@ -490,6 +490,29 @@ After execution, the resulting values of the `output` and `input-output` paramet
 
+Database connector supports the use of callable statements with the format { call procedureName(:param1, :param2, ..., :paramN) } where parameters will be matched positionally, that is to say, :paramN will match the Nth parameter in the stored procedure declaration. Suppose you have declared a stored procedure which receives two parameters: myFirstParameter and mySecondParameter, in that order. T ...
+
+It also does not support the use of engine specific embedded functions, for example, you might want to change a date to a specific format before calling a procedure in an Oracle database, however, the following approach will not work:  { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }. All data transformations should be done using dataweave, before the invocation to ...
```
wrap { call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }
as code snippets `{ call myStoredProcedure(paramName => to_date(:param1, 'YYYY-MM-DD HH:mm:ss')) }`

kevintroller replied:
Updated.

mojavelinux committed:
update connectors metadata

- add missing exchange URLs
- add missing release notes xrefs
- add or fix support levels
- fix inconsistent titles
- fix inconsistent descriptions

Commit:
Update amazon-sns-connector-reference.adoc

kevintroller committed:
Merge pull request #654 from mulesoft/aws-sns/release-4_4_2

Update amazon-sns-connector-reference.adoc

dhirsch1 committed:
Added doc edits

dhirsch1 committed:
Commit before release

dhirsch1 committed:
Merge pull request #635 from mulesoft/CONN-4558

Added the migration guide for Salesforce Marketing Cloud 3.x

dhirsch1 committed:
Interim commit

dhirsch1 committed:
More edits

dhirsch1 committed:
Merge pull request #646 from mulesoft/aws-s3/release-5_6_0

Amazon s3 - Release 5.6.0 - M4

dhirsch1 committed:
More cleanup work

dhirsch1 committed:
Commit before release

Commit:
SE-14978 | Fixed documentation for the connector

alejandrodifiore committed:
Update ms-service-bus-connector.adoc

 Spelling error correction

Commit:
SE-14978 | Fixed documentation for the connector

dhirsch1 committed:
Merge pull request #656 from mulesoft/Amazon-S3-Ref-5-6-0-DH

Amazon S3 Connector Reference 5.6.0

dhirsch1 committed:
Merge pull request #657 from mulesoft/SE-14978

SE-14978 | Fixed documentation for the connector

dhirsch1 committed:
One fix

dhirsch1 committed:
Merge pull request #611 from mulesoft/DOCS-edit-salesforce-design-ctr-DH

Salesforce Connector Design Center Edit

valkyrie69 committed:
Updates for version 4.0.0

valkyrie69 committed:
Adding version 3.0 reference

mojavelinux committed:
fix metadata and invalid xrefs

mojavelinux committed:
fix missing entries and xrefs for jms connector

dhirsch1 committed:
Entered Kim's edits

dhirsch1 committed:
Merge pull request #614 from mulesoft/DOCS-edit-salesforce-studio-DH

Salesforce Connector Studio Topic to Edit

dhirsch1 committed:
Added Kim's edits

valkyrie69 committed:
Updates to changes information

mojavelinux committed:
fix obsolete and broken image refs

dhirsch1 committed:
Entered Kim's edits

dhirsch1 committed:
Revert "Entered Kim's edits"

This reverts commit 46a80c15b83362ed36f2d26f7bd599731af4b119.

dhirsch1 committed:
Entered Kim's edits for the XML Maven topic

dhirsch1 committed:
Added back Kim's edits to the XML Maven file

dhirsch1 committed:
Added back the examples file, which I had accidentally deleted.

dhirsch1 committed:
Added suggested edits from another PR.

dhirsch1 committed:
Merge pull request #629 from mulesoft/DOCS-edit-Salesforce-examples-edit-DH

Salesforce Connector XML Topic for Edit

dhirsch1 committed:
Added Kim's edits

Commit:
Oauth update

Commit:
reference adoc updated

dhirsch1 committed:
Added missed Valkyrie's edits

dhirsch1 committed:
Added comments from Valkyrie that I missed. Made one correction.

dhirsch1 committed:
One more correction

dhirsch1 committed:
Removed a period

dhirsch1 committed:
Merge pull request #621 from mulesoft/DOCS-Salesforce-Connector-Upgrade-Edit

Salesforce Connector v10 Upgrade and Migrate Topic for Edit

dhirsch1 committed:
Added Kim's edits

dhirsch1 committed:
Found more edits to add

dhirsch1 committed:
Correct small errors

dhirsch1 committed:
Merge pull request #620 from mulesoft/docs-salesforce-connector-landing-page

Salesforce Connector v10 Landing Page Standards Edit

dhirsch1 committed:
Added small edit by Valkyrie

dhirsch1 committed:
Merge pull request #660 from mulesoft/Fix-Salesforce-Studio-DH

Salesforce Connector Studio - Comments from Valkyrie and one correction

dhirsch1 committed:
Merge pull request #659 from mulesoft/Fix-Salesforce-Design-Center-DH

Added missed edits for Salesforce Connector Design Center topic

dhirsch1 committed:
Merge pull request #622 from mulesoft/DOCS-edit-Salesforce-Config-DH-Edit

Salesforce Connector Additional Config Topic Edit

valkyrie69 committed:
Removed Describe operation

Commit:
CONN-14978 | Fixed xml

dhirsch1 committed:
Merge pull request #661 from mulesoft/CONN-14978

CONN-14978 | Fixed xml for the demo.

dhirsch1 committed:
Interim commit of graphics

fsgonz committed:
Update amqp-documentation.adoc

valkyrie69 committed:
Updates for javier

And regenerated reference

valkyrie69 committed:
Updated image

mcherem committed:
Update as2-connectorMule4.adoc

Doc edits.

mcherem committed:
Update AS2-connector-examples.adoc

Doc edits.

mcherem committed:
Update AS2-connector-examples.adoc

Minor doc edits.

mcherem committed:
Update as2-connector-reference.adoc

Doc edits.

mcherem committed:
Update as2-connector-reference.adoc

Doc edits.

valkyrie69 committed:
Updates

valkyrie69 committed:
Updated images

mojavelinux committed:
move exchange links to see also; remove "The" in front of module name

mcherem committed:
Update as2-connector-reference.adoc

Doc edits.

mcherem committed:
Update as2-connector-reference.adoc

Doc edits.

xidex committed:
Update servicenow-connector.adoc

Commit:
Update sap-s4hana-cloud-connector-reference.adoc

valkyrie69 committed:
Updates for 1.6

valkyrie69 committed:
Merge pull request #662 from mulesoft/fsgonz-patch-3-1

Update amqp-documentation.adoc

dhirsch1 committed:
Commit of fixes related to the Update operations operation.

hannanelson committed:
Addressed input from #643

hannanelson committed:
Addressed inpub from #644

alejandrodifiore committed:
Update ms-service-bus-connector.adoc

SAS token provider implementation update.

alejandrodifiore committed:
Update ms-service-bus-connector.adoc

Improvements in the explanation

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Added fixes for two bugs (DOCS-7188 and DOCS-7189) and added links to the intro to connectors topic.

dariohvega committed:
Update ms-service-bus-connector.adoc

dariohvega committed:
CONN-4658 | Updated Images

valkyrie69 committed:
Diane edits Added upgrade topic to nav

valkyrie69 committed:
Merge pull request #652 from mulesoft/DOCS-7260-kafka4.0-edit

Docs 7260 Kafka 4.0

dhirsch1 committed:
Tweaked the wording for CØ˜˜-4658

dhirsch1 committed:
Merge pull request #650 from mulesoft/SE-14685

[DO NOT MERGE]SE-14685 | Fixed SasTokenProvider interface

dhirsch1 committed:
Interim commit of graphics

dhirsch1 committed:
Interim commit

valkyrie69 committed:
fixed typos

valkyrie69 committed:
Merge pull request #668 from mulesoft/DOCS-7260-kafka4.0-edit

fixed typos

kevintroller committed:
Editorial updates

dhirsch1 committed:
Merge pull request #667 from mulesoft/DOCS-Two-Salesforce-10-Bugs-DH

Fixed two doc bugs for Salesforce Connector v10.0

hannanelson committed:
Update object-store-connector.adoc

valkyrie69 committed:
ServiceNow 6.5.0

hannanelson committed:
Update modules/ROOT/pages/object-store/object-store-connector.adoc

Co-Authored-By: Will Warner <45182827+wwarner-salesforce@users.noreply.github.com>

hannanelson committed:
Merge pull request #665 from mulesoft/DOCS-7396-OS-Incorporate-contributed-bugfixes-hn

Docs 7396 os incorporate contributed bugfixes hn

luizamujdei committed:
added details for object store usage SF 9x

gabrieldalborgo committed:
Update compression-documentation.adoc

gabrieldalborgo committed:
Update compression-documentation.adoc

dhirsch1 committed:
Interim Commit

dhirsch1 committed:
Update netsuite-connector-reference.adoc

dhirsch1 committed:
Interim commit

valkyrie69 committed:
Updates for ServiceNow 6.5.0

valkyrie69 committed:
Edits

sarathecoubian committed:
note for no spaces

valkyrie69 committed:
Kevin edits

valkyrie69 committed:
Merge pull request #648 from mulesoft/servicenow/release-6_5_0

ServiceNow - release 6.5.0 - M4

Commit:
SE-14816 | Remove filters from the documentation Windows service bus

kevintroller committed:
Minor version fix.

dhirsch1 committed:
Update netsuite-reference.adoc

sarathecoubian committed:
Merge pull request #672 from mulesoft/DOCS-7478-Custom-Metrics-connector;-Dimension-Name-and-Fact-Name-can't-contain-spaces

note for no spaces

gabrieldalborgo committed:
Merge pull request #670 from mulesoft/gabrieldalborgo-patch-1

Compression Module Documentation Reference v2.1

valkyrie69 committed:
Update jms-connector.adoc

valkyrie69 committed:
Update jms-consume.adoc

valkyrie69 committed:
Update jms-activemq-configuration.adoc

valkyrie69 committed:
Update include_link_list.adoc

valkyrie69 committed:
Update jms-connector.adoc

valkyrie69 committed:
Update jms-consume.adoc

dhirsch1 committed:
Committing a fix and removal of a non-needed file.

valkyrie69 committed:
Update include_link_list.adoc

valkyrie69 committed:
Update jms-activemq-configuration.adoc

valkyrie69 committed:
Merge branch 'latest' into update-connectors-metadata

valkyrie69 committed:
Merge pull request #658 from mulesoft/update-connectors-metadata

update metadata for connectors

dhirsch1 committed:
Read for review

dhirsch1 committed:
Added periods to two descriptions.

dhirsch1 committed:
Fixed typo.

kevintroller committed:
Merge pull request #674 from mulesoft/DOCS-7537-CONN-Compression-Module-2.1-kt

DOCS-7537-CONN-Compression-Module-2.1-kt

dhirsch1 committed:
Shortened URL to make it render better.

dhirsch1 committed:
Merge pull request #673 from mulesoft/SE-14816

[DO NOT MERGE PENDING REVIEW]SE-14816 | Remove filters from the documentation Windows service bus

dhirsch1 committed:
Removed unsupported authentication types

dhirsch1 committed:
Added link to Exchange

dariohvega committed:
CONN-4658 | Replaced Azure word in doc and xml. Flow names updated

fermujica committed:
Add necessary page alias

fermujica committed:
Merge branch 'latest' into DOCS-6491-NetSuite-EndPointURL-DH

fermujica committed:
Remove netsuite-connector-reference

dhirsch1 committed:
Merge pull request #675 from mulesoft/DOCS-6491-NetSuite-EndPointURL-DH

Added a better description to Web Service Endpoint URL field to fix a bug

dhirsch1 committed:
Interim commit of graphics

dariohvega committed:
Merge branch 'latest' into CONN-4658

dhirsch1 committed:
Merge pull request #666 from mulesoft/CONN-4658

CONN-4658 | Update ms-service-bus-connector.adoc

dhirsch1 committed:
Interim commit of graphics

dhirsch1 committed:
Interim commit of graphics

dhirsch1 committed:
Worked on captions and alt text.

valkyrie69 committed:
Updated Kafka Connector 3.0

dhirsch1 committed:
Interim commit of graphics

valkyrie69 committed:
Diane comments

valkyrie69 committed:
Merge pull request #677 from mulesoft/update-kafka-3.0-reference

DOCS-7569 Updated Kafka Connector 3.0

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Got a screenshot to appear, fixed some wording.

dhirsch1 committed:
Preparing for a doc edit

dhirsch1 committed:
Merge branch 'latest' into DOCS-6912-config-studio-DH

dhirsch1 committed:
Removed extra png files

dhirsch1 committed:
Fixed the nav file

dhirsch1 committed:
Changed arrows in some graphics

dhirsch1 committed:
Fixed graphic

mcherem committed:
Update as2-connectorMule4.adoc

mcherem committed:
Merge branch 'latest' into DOCS-7047-AS2-5.1-UG-updates-mc

mcherem committed:
Merge pull request #573 from mulesoft/DOCS-7047-AS2-5.1-UG-updates-mc

Update as2-connectorMule4.adoc

kevintroller committed:
Peer editorial updates

kevintroller committed:
Merge pull request #617 from mulesoft/update-ldap-example-parameter-names

Fixed incorrect parameter name

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Interim commit

dariohvega committed:
CONN-4647 | Added document with examples for Search Operation and update references

dhirsch1 committed:
Interim commit

sebasgold committed:
Add the insecure check for each connection and the usage of the secure connections in each case.

Add the insecure check for each connection and the usage of the secure connections in each case.

Commit:
MULE-17949: Added example showing how to use additional-properties element.

kevintroller committed:
Minor wording

kevintroller committed:
Merge pull request #681 from mulesoft/sebasgold-patch-2

Add the insecure check for each connection and the usage of the secur…

kevintroller committed:
Update version

dhirsch1 committed:
Interim commit

Commit:
ref guide api update

valkyrie69 committed:
Edits

valkyrie69 committed:
Updates

valkyrie69 committed:
Updates to user guide

valkyrie69 committed:
Update sap-s4hana-cloud-connector.adoc

dhirsch1 committed:
Interim commit

kevintroller committed:
Merge pull request #682 from mulesoft/MULE-17949

MULE-17949: Oracle Data Source Config doesn't allow for additional po…

valkyrie69 committed:
Added OAuth info

And other edits

valkyrie69 committed:
Edits

dhirsch1 committed:
Interim commit

valkyrie69 committed:
Minor edit

dhirsch1 committed:
Update salesforce-connector-95.adoc

Interim commit

valkyrie69 committed:
Added Upgrade topic

valkyrie69 committed:
Update sap-s4hana-cloud-connector.adoc

valkyrie69 committed:
Renamed upgrade file

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Commit of changes for v9.5 before technical edit

kevintroller committed:
SMTPS guidelines

valkyrie69 committed:
Added some upgrade info

dhirsch1 committed:
Interim commit

Commit:
doc update

valkyrie69 committed:
Minor edits

valkyrie69 committed:
Updated title

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-s4hana-cloud-connector-upgrade-migrate.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Merge pull request #663 from mulesoft/sap-s4hana-cloud/release-2.0.0

Update sap-s4hana-cloud-connector-reference.adoc

valkyrie69 committed:
Revert "Update sap-s4hana-cloud-connector-reference.adoc"

valkyrie69 committed:
Merge pull request #685 from mulesoft/revert-663-sap-s4hana-cloud/release-2.0.0

Revert "Update sap-s4hana-cloud-connector-reference.adoc"

dhirsch1 committed:
Iterim commit

valkyrie69 committed:
Revert "Revert "Update sap-s4hana-cloud-connector-reference.adoc""

valkyrie69 committed:
Update sap-s4hana-cloud-connector-upgrade-migrate.adoc

valkyrie69 committed:
Update sap-s4hana-cloud-connector-upgrade-migrate.adoc

valkyrie69 committed:
Update sap-s4hana-cloud-connector.adoc

valkyrie69 committed:
Merge pull request #687 from mulesoft/revert-685-revert-663-sap-s4hana-cloud/release-2.0.0

Revert "Revert "Update sap-s4hana-cloud-connector-reference.adoc""

dhirsch1 committed:
Added prepared graphics

dhirsch1 committed:
Completed update of the graphcis

dhirsch1 committed:
Commit before technical edit

dhirsch1 committed:
Added Martin's fixes

kevintroller committed:
Editorial

kevintroller committed:
Merge pull request #684 from mulesoft/DOCS-5481-SMTPS-Gmail-update-kt

DOCS-5481-SMTPS-Gmail-update-kt

dhirsch1 committed:
Commit before doc edit

dhirsch1 committed:
Fix to Nav for Studio topic being under the NetSuite OpenAir Connector

valkyrie69 committed:
Updated Note to add RTF and PCE

valkyrie69 committed:
Moved Terminology to a new topic

dhirsch1 committed:
Interim commit of doc edits

dhirsch1 committed:
Commit with edits incorporated

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Interim commit

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Update modules/ROOT/pages/sap/sap-connector-terminology.adoc

Co-Authored-By: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

valkyrie69 committed:
Kim edits

valkyrie69 committed:
Merge pull request #689 from mulesoft/DOCS-7458-add-note-snc

Updated Note to add RTF and PCE

dhirsch1 committed:
Prepared object store information for release

dhirsch1 committed:
Merge pull request #669 from mulesoft/sf9-userManual

added details for object store usage SF 9x

dhirsch1 committed:
Added final fixes and screenshots

dhirsch1 committed:
Merge pull request #679 from mulesoft/CONN-4647

CONN-4647 | Added document with examples for Search Operation and upd…

dhirsch1 committed:
Last commit before merge

dhirsch1 committed:
Merge branch 'latest' into DOCS-6912-config-studio-DH

dhirsch1 committed:
Merge pull request #546 from mulesoft/DOCS-6912-config-studio-DH

Connector Configuration Overview and Studio Topics

dhirsch1 committed:
Interim commit

dhirsch1 committed:
Added doc edits

dhirsch1 committed:
Merge pull request #692 from mulesoft/DOCS-7673-Microsoft-Dynamics-User-DH

Updated Microsoft Dynamics 365 for Ops Reference Guide

kevintroller committed:
Minor wording

kevintroller committed:
Minor wording

Commit:
Calling Oracle Stored Procedure with TO_DATE() by DB Connector 1.5.5 aborts 'Timestamp format must be yyyy-mm-dd hh:mm:ss[.fffffffff]'

kevintroller commented:
Closing this PR to avoid conflict during merge. New PR https://github.com/mulesoft/docs-connectors/pull/696/files
